_jugs/DelhiJUG.md

```diff
@@ -0,0 +1,8 @@
+---
+name:     "Delhi NCR Java User Group"
+country:  India
+website:  https://jugdelhi.org/, https://www.meetup.com/Delhi-NCR-JUG/
```

Comment:
The website right now only accepts 1 link 😅
It is the link that will appear on the map. 
Can you please just use only one?

Reply:
@icougil  took care of it. Thanks!

Reply:
cool, thank you !